Assets/00_Content/Scripts/Vikings/VikingData.cs

```diff
@@ -29,5 +29,20 @@ public class VikingData : ScriptableObject {
 
 		[Tooltip("How long does it take to perform one attack, s/attack")]
 		public float attackRate = 1;
+
+		[Tooltip("How many hits can be taken when brawling")]
+		public float brawlHealth = 3;
+
+		public float attackTriggerDistance = 4f;
+
+		public float spinAttackSpeed = 8f;
+
+		public float spinAttackDuration = 2f;
+
+		public float attackDamage = 1f;
+
+
+		[Header("Debug")]
+		public bool defaultAttackPlayer;
```

Comment:
What does this do?

Reply:
Who knows... really bad naming and im going to add a comment 👍